src/titiler/application/titiler/application/main.py

```diff
@@ -215,7 +217,10 @@ def validate_access_token(access_token: str = Security(api_key_query)):
 )
 def ping():
     """Health check."""
-    return {"ping": "pong!"}
+    with redirect_stdout(io.StringIO()) as f:
+        show_versions()
+    rasterio_versions = f.getvalue().splitlines()
+    return {"Hello from titiler!": rasterio_versions}
```

Comment:
This is admittedly trickier than I expected! I was wondering if it would be better to just use rasterio._get_deps_info() even though it's a private method because the python dictionary would be nicer than capturing a multiline string 😅

Reply:
👍 we could maybe simply show 

https://github.com/rasterio/rasterio/blob/64474c1795d5d47c03e271386d21c3b244769946/rasterio/__init__.py#L85-L88

```python
def ping():
     """Health check."""
     return {
          "versions": {
               "titiler": titiler_version,
               "rasterio": rasterio.__version__,
               "gdal": rasterio.__gdal_version__,
               "proj": rasterio.__proj_version__,
               "geos": rasterio.__geos_version__,
          }
     }
```

Reply:
Agreed that is better! The GEOS version oddly does not seem correct, but I guess that's an upstream issue...

```python
{
  "versions": {
    "titiler": "0.19.2",
    "rasterio": "1.4.3",
    "gdal": "3.9.3",
    "proj": "9.4.1",
    "geos": "0.0.0"
  }
}
```